clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -49,22 +49,21 @@ export class LanguageClientService {
     private createClientOptions(): LanguageClientOptions {
         const signatureFunc: ConfigurationRequest.MiddlewareSignature = async (
             params: ConfigurationParams,
-            token: CancellationToken,
+            token: vscode.CancellationToken,
             next: ConfigurationRequest.HandlerSignature) => {
-
-            // TODO if request params are right
-            return (await this.copybooksPathGenerator.listUris()).map(uri => uri.toString());
-            // TODO else return next(params, token);
+                return await this.middleware.handleConfigurationRequest(params, token, next);
```

Comment:
The line indent not adjusted

Reply:
I don't see any problem about indentation in my editor